cf.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "------ Create database tables ------"
+
+python manage.py migrate --noinput
+python manage.py load_agency_contacts
```

Comment:
We need to run 'python manage.py collectstatic' here. We should at least add a commented out line (as a placeholder) while we figure out all the S3 stuff.